deltachat-rpc-client/tests/test_something.py

```diff
@@ -286,6 +286,43 @@ def test_message(acfactory) -> None:
     assert reactions == snapshot.reactions
 
 
+def test_reaction_seen_on_another_dev(acfactory, tmp_path) -> None:
```

Comment:
It fails, though for some reason i don't see the test log: https://github.com/deltachat/deltachat-core-rust/actions/runs/11978349803/job/33398412674?pr=6246. Maybe because i cancelled the workflow